source/draw/RasterYuv420Semiplanar.ooc

```diff
@@ -24,9 +23,26 @@ import io/FileReader
 import io/Reader
 import io/FileWriter
 
-RasterYuv420Semiplanar: class extends RasterYuvSemiplanar {
+RasterYuv420Semiplanar: class extends RasterImage {
+	_y: RasterMonochrome
+	_uv: RasterUv
+	y ::= this _y
+	uv ::= this _uv
 	stride ::= this _y stride
-	init: func ~fromRasterImages (yImage: RasterMonochrome, uvImage: RasterUv) { super(yImage, uvImage) }
+	init: func ~fromRasterImages (yImage: RasterMonochrome, uvImage: RasterUv) {
```

Comment:
Also maybe `(this_y, this _uv) referenceCount increase()` would save some space.